spec/requests/static_requests_spec.rb

```diff
@@ -51,13 +55,14 @@
 
   describe "Super user without org signed in" do
     before do
-      sign_in(@super_admin_no_org)
+      sign_in(create(:super_admin, organization: nil))
     end
 
     describe "GET #index" do
       it "redirects to admin dashboard" do
         get root_path
-        expect(response).to redirect_to(admin_dashboard_url(@admin))
+
+        expect(response).to redirect_to(admin_dashboard_url)
```

Comment:
so..... `@admin` does not actually exist. `@admin == nil` thus, `admin_dashboard_url(@admin)` is the same as `admin_dashboard_url`